### DISCUSSION TITLE
关于LLMs-from-scratch项目中文版本合作的提议

### DISCUSSION THREAD

**Discussion by Ethan-Chen-plus** [Intelligence-Manifesto/LLMs-from-scratch#3] [General]
尊敬的Intelligence-Manifesto团队，

您好！我是来自datawhale-china开源组织的Ethan-Chen-Plus，我们是一个致力于推动开源项目和数据科学教育的非盈利组织。最近，我们注意到贵组织正在进行LLMs-from-scratch项目的开发，这是一个我们非常感兴趣并且已经开始着手进行中文版本开发的项目，我们目前的进展可以从如下链接看到：https://[github.com/datawhalechina/llms-from-scratch-cn](https://github.com/datawhalechina/llms-from-scratch-cn)。

datawhale-china拥有一支强大的开发者和翻译者团队，我们相信通过我们的合作，可以加速LLMs-from-scratch项目中文版本的开发和推广，为中文读者提供高质量的教育资源。此外，我们认为合作不仅能增加项目的可见度，还能促进开源社区成员之间的知识分享和技术进步。

我们希望能够与Intelligence-Manifesto团队一起完成这个开源项目。如果您对此合作提议感兴趣，请通过kewei_chen@hust.edu.cn或者直接在该讨论区下方与我联系，以便我们进一步讨论具体细节。

期待您的回复。

最诚挚的问候，

Ethan-Chen-Plus
datawhale-china开源组织

---

Dear Intelligence-Manifesto Team,

Greetings! I am Ethan-Chen-Plus from datawhale-china, a nonprofit organization dedicated to promoting open-source projects and data science education. Recently, we noticed that your organization is developing the LLMs-from-scratch project, which is a project we are very interested in and have already begun developing a Chinese version. Our current progress can be seen from the following link: [https://github.com/datawhalechina/llms-from-scratch-cn](https://github.com/datawhalechina/llms-from-scratch-cn).

Datawhale-china boasts a strong team of developers and translators. We believe that through our collaboration, we can accelerate the development and promotion of the Chinese version of the LLMs-from-scratch project, providing high-quality educational resources to Chinese readers. Additionally, we believe that collaboration will not only increase the visibility of the project but also promote knowledge sharing and technological advancement among members of the open-source community.

We hope to work together with the Intelligence-Manifesto team to complete this open-source project. If you are interested in this collaboration proposal, please contact me at kewei_chen@hust.edu.cn or directly reply to this discussion for further discussion on specific details.

Looking forward to your reply.

Best regards,

Ethan-Chen-Plus
Datawhale-china Open Source Organization

**Comment by Intelligence-Manifesto** [comment]
当然可以我们的团队可以参与合作，很开心
